app/views/govuk_publishing_components/components/_fieldset.html.erb

```diff
@@ -1,6 +1,6 @@
 <% text = text || yield %>
-<fieldset class="gem-c-fieldset">
-  <legend class="gem-c-fieldset__legend">
+<fieldset class="gem-c-fieldset govuk-fieldset">
```

Comment:
At some point we should talk about the merit of keeping `gem-c-` classes on components if they don't actually style anything. Although I guess it's needed for the browser extension to recognise them?

Reply:
Yep, we currently only keep them for the browser extension. We could think about removing them in-sync with an update on the extension.